lib/layout/text_scale.dart

```diff
@@ -8,8 +8,8 @@ import 'package:flutter/material.dart';
 
 import 'package:gallery/data/gallery_options.dart';
 
-double _textScaleFactor(BuildContext context) {
-  return GalleryOptions.of(context).textScaleFactor(context);
+double? _textScaleFactor(BuildContext context) {
```

Comment:
Let's leave this file as is for now until `GalleryOptions` is migrated

Reply:
Sure , there is also analyzer issue, i will recheck on my end , I ran the tests but missed analyzer . Will make sure to run on all future pr's.